lib/cart/cart_repository.dart

```diff
@@ -101,4 +104,32 @@ class CartRepository {
       return false;
     }
   }
+
+  Future<void> changeQuantityInCart({
+    required int entryNumber,
+    required int quantity,
+  }) async {
+    const cartCode = kTestCart;
+
+    try {
+      await _cartApi.changeQuantityInCart(
+        cartCode: cartCode,
+        entryNumber: entryNumber,
+        quantity: quantity,
+      );
+
+      // TODO: add any applicable cart messages
+      return;
+    } catch (e) {
```

Comment:
The strategy I have setup when writing the category bloc and implementing crashlytics is trycatching the `DioException` only and eventually other likely other that could happen like "didn't add because product is out of stock", but let any other error bubbling up and got catch by Crashlytics and logged in the terminal.